daml-lf/spec/daml-lf-1.rst

```diff
@@ -1887,140 +1887,123 @@ will always be used to compare values of same types::
 .. note:: the equality of generic map is not sensitive to the order of
           its entries. See rules ``'GenEqNonEmptyGenMap'``.
 
-Value order
+Type order
```

Comment:
You are right, this is confusing. 
This section is about *only* type ordering. 
The main difference between type ordering and value ordering is that we need to specify the latter *operationally* to explain how it fails when encounter non comparable values. We do not need an operation specification for the former because the type checker will ensure that comparison of is always defined (the rules for typing ``'to_any'`` and ``'type_rep'``  ensure the type used do not contains variable, quanitifier not type synonyms).


Value ordering is define in the section ``Generic comparison functions``.